docs/content/preview/explore/query-1-performance/pg-stat-progress-copy.md

```diff
@@ -0,0 +1,174 @@
+
+This is required for YugabyteDB, because if the `COPY` command finishes due to an error, then you would like to know how many rows were reliably persisted to the disk.
+
+## Examples
+
+The following examples demonstrates the possible stages (IN PROGRESS, ERROR, SUCCESS) for the copy operation.
+
+### Prerequisites
+
+- Make sure you have YugabyteDB up and running. If you are new to YugabyteDB, follow the steps in [Quick start](../../../quick-start/) to have YugabyteDB up and running in minutes.
+
+- Run the following script from your YugabyteDB installation directory to generate data to a file `test_data.csv`.
+
+    ```sh
+    #!/bin/bash
+    rm -rf test_data.csv
+    for i in {1..100000}
+    do
+       echo "$i, $i, $i" >> test_data.csv
+    done
+    ```
+
+- From your local YugabyteDB installation directory, connect to the [YSQL](../../../../admin/ysqlsh/) shell, and create a table using the following schema:
+
+    ```sql
+    create table test_copy ( h1 int, r1 int, v1 int, primary key (h1, r1));
+    ```
+
+### Run a successful COPY command
+
+1. Copy `test_data.csv` using the following copy command:
+
+    ```sql
+    \copy test_copy from 'test_data.csv'  (DELIMITER ',');
+    ```
+
+    ```output
+    COPY 100000
+    ```
+
+1. Verify `yb_status` using the view.
+
+    ```sql
+    select * from pg_stat_progress_copy ;
+    ```
+
+    You should see output similar to the following:
+
+    ```output
+     pid  | datid | datname  | relid |  command  | type | yb_status | bytes_processed | bytes_total | tuples_processed | tuples_excluded
+    -------+-------+----------+-------+-----------+------+-----------+-----------------+-------------+------------------+-----------------
+    74390 | 13288 | yugabyte | 16394 | COPY FROM | PIPE | SUCCESS   |         1966685 |           0 |           100000 |               0
+    (1 row)
+    ```
+
+### Verify a COPY command status in progress
+
+Copy `test_data.csv` using the following copy command and check the status in parallel from another terminal.
+
+```sql
+\copy test_copy from 'test_data.csv'  (DELIMITER ',');
+```
+
+```sql
+select * from pg_stat_progress_copy ;
+```
+
+You should see output similar to the following:
+
+```output
+ pid  | datid | datname  | relid |  command  | type |  yb_status  | bytes_processed | bytes_total | tuples_processed | tuples_excluded
+-------+-------+----------+-------+-----------+------+-------------+-----------------+-------------+------------------+-----------------
+74390 | 13288 | yugabyte | 16404 | COPY FROM | PIPE | IN PROGRESS |          766682 |           0 |            40000 |               0
+(1 row)
+```
+
+### Verify an interrupted COPY operation
+
+1. Copy `test_data.csv` and interrupt it using the following commands:
+
+    ```sh
+    \copy test_copy from 'test_data.csv'  (DELIMITER ',');
+    ```
+
+    ```output
+    ^CCancel request sent
```

Comment:
CCancel - maybe separate the ^C from 'Cancel request sent' for clarity. I assume it's Ctrl+C? maybe spell that out in the step